core/src/main/java/org/jruby/RubyHash.java

```diff
@@ -62,13 +62,7 @@
 import org.jruby.util.TypeConverter;
 
 import java.io.IOException;
-import java.util.AbstractCollection;
-import java.util.AbstractSet;
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.Map;
-import java.util.NoSuchElementException;
-import java.util.Set;
+import java.util.*;
```

Comment:
Oh missed this one.  we do not star up packages.  This is probably a setting in your IDE doing this I bet.